src/pages/main/tools/Editor/Editor.tsx

```diff
@@ -0,0 +1,68 @@
+import Draggable, { DraggableData, DraggableEvent } from 'react-draggable';
+import { LatexTextArea, PdfViewer } from '@components';
+
+import { RxDotsVertical } from 'react-icons/rx';
+import styles from './Editor.module.less';
+import { useResizeDetector } from 'react-resize-detector';
+import { useState } from 'react';
+
+const Editor = () => {
```

Comment:
no need for directory `tools` in `pages/main`, just place it in `pages/main/editor/Editor.tsx`